src/System.Memory/src/System/Buffers/BuffersExtensions.cs

```diff
@@ -36,12 +56,35 @@ public static class BuffersExtensions
         public static void CopyTo<T>(this ReadOnlySequence<T> sequence, Span<T> destination)
         {
             if (sequence.Length > destination.Length)
+            {
                 ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.destination);
+            }
```

Comment:
Stop lowering code coverage :) Closing brace counts as non covered line when `throw` is not explicit 🤣

Reply:
Removed